### PR TITLE
Update game

### PR DESCRIPTION
**Pull request by PoojaSharma07**
test description.

### CODE REVIEW

**Review by PoojaSharma07 — COMMENTED** [review]
test review.

File: index.html
Comment (v-PoojaSharma07) [inline index.html:1]: test comment

Comment (v-PoojaSharma07) [reply index.html:1]: suggested change

**Review by PoojaSharma07 — COMMENTED** [review]
testing

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
 
 <script>
```
Comment (v-PoojaSharma07) [inline index.html:52]:
```suggestion
<script>
test
```

**Pull request merged**